### DISCUSSION TITLE
Problem with class-level bracket matching in java files

### DISCUSSION THREAD

**Discussion by skytwosea** [facelessuser/BracketHighlighter#616] [Q&A] [answered]
As my java files grow in complexity, the class-scope bracket fails to match; the gutter contains a question mark at both the uppermost and lowermost bracket. Is this a known issue with java files?
Registered Stable build 4143 on linux 5.15.0-73-generic

**Comment by facelessuser** [comment, marked as answer]
This should probably explain it: https://github.com/facelessuser/BracketHighlighter/issues/422#issuecomment-314506438

**Reply by skytwosea** [reply]
Hey, thanks a ton: ignore_threshold did the trick.
Much appreciated!
